tests/frontend/test_ratings.py

```diff
@@ -502,3 +504,151 @@ def test_delete_developer_reply_to_review(selenium, base_url, variables):
             (By.CSS_SELECTOR, '.AddonReviewCard-reply .ShowMoreCard-contents > div')
         )
     )
+
+
+@pytest.mark.nondestructive
+def test_rating_card_loaded_correctly(selenium, base_url, variables):
+    selenium.get(variables["addon_version_page_url"])
+    page = Versions(selenium, base_url)
+    rating_card = page.rating_card
+    assert rating_card.addon_image.is_displayed()
+    assert rating_card.addon_title.is_displayed()
+    assert len(rating_card.addon_author_names) >= 1
+    for star in rating_card.rating_stars:
+        assert star.is_displayed()
+    assert rating_card.rating in range(6)
+    for i in range(5):
+        assert page.rating_card.number_of_reviews_with_specific_stars(i) >= 0
+    for rating_bar in rating_card.rating_bars:
+        assert rating_bar.is_displayed
+
+
+@pytest.mark.nondestructive
+def test_rating_card_title(selenium, base_url, variables):
+    selenium.get(variables["addon_version_page_url"])
+    page = Versions(selenium, base_url)
+    # get title from rating card
+    addon_title_from_card = page.rating_card.addon_title.text
+    # go to addon detail page
+    addon_detail_name = page.rating_card.click_addon_title()
+    # verify if title matches
+    assert addon_title_from_card == addon_detail_name.name
+
+
+@pytest.mark.nondestructive
+def test_rating_card_authors(selenium, base_url, variables):
+    selenium.get(f'{base_url}/addon/{variables["addon_with_stats"]}/versions/')
+    page = Versions(selenium, base_url)
+    addon_title = page.rating_card.addon_title.text
+    # iterate through the authors
+    for i in range(len(page.rating_card.addon_author_names)):
+        # go to author's page
+        user_page = page.rating_card.click_author_name(i)
+        # make a list of author's extensions
+        extensions = []
+        for item in user_page.user_extensions.result_list.extensions:
+            extensions.append(item.name)
+        # verify if the current add-on is in the list
+        assert addon_title in extensions
+        # go back for testing the next author
+        page.driver.back()
+
+
+@pytest.mark.nondestructive
+@pytest.mark.parametrize(
+    'addon_slug',
+    ('tabliss', 'facebook-container', 'bitwarden-password-manager', 'unloadtabs'),
+    ids=(
+        '5 stars rating add-on',
+        '4.3 stars rating add-on',
+        '4 stars rating add-on',
+        '3.5 stars rating add-on',
+    ),
+)
```

Comment:
There is no guarantee that these addons will have these ratings and the slugs might also change, which will make the add-ons inaccessible. They are also hardcoded to addons present on stage. This test, in it's current format, is not durable. 

You should try a different method if you want to test the rating stars.

Reply:
Changed it :)